openquake/hazardlib/geo/surface/base.py

```diff
@@ -427,6 +427,41 @@ def get_middle_point(self):
 
         return mesh.get_middle_point()
 
+    def get_resampled_top_edge(self, angle_var=0.1):
```

Comment:
The output, giving all the vertices coordinate of the fault surface is not used anymore.